weather/www/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    <title></title>
+
+    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+
+    <!-- IF using Sass (run gulp sass first), then uncomment below and remove the CSS includes above
+    <link href="css/ionic.app.css" rel="stylesheet">
+    -->
+
+    <!-- ionic/angularjs js -->
+    <script src="lib/ionic/js/ionic.bundle.js"></script>
+	<script src="lib/ionic/js/angular/angular-resource.js"></script>
+
+    <!-- cordova script (this will be a 404 during development) -->
+    <script src="cordova.js"></script>
+
+    <!-- your app's js -->
+    <script src="js/app.js"></script>
+	<script src="js/weatherController.js"></script>
+	<script src="js/services.js"></script>
+	<script src="js/constants.js"></script>
+  </head>
+  
+  <!--Header stuff -->
+  <body ng-app="weather">
+	<ion-nav-view class="background">
+	<ion-header-bar class="bar-calm">
+		<h1 class="title">Wenham, MA</h1>
```

Comment:
It would be cool to have the title bound to the Angular JS, so it can be dynamically set!